package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "Restaurant-Page",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"deploy-demo": "git push origin :gh-pages && git subtree push --prefix dist origin gh-pages"
+	},
```

Comment:
### [required change]
- Add Necessary run common commands like build, development should be in **package.json** inside `scripts'. [ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png)]

``` 

"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" 
  },

```

The `start` option is OPTIONAL in this project

Reply:
### [optional change]
- You can add my gif screenshot of your final display inside your README.....it's more visually self-explanatory